staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -188,8 +188,48 @@ type VirtualMachineInstanceStatus struct {
 	// ActivePods is a mapping of pod UID to node name.
 	// It is possible for multiple pods to be running for a single VMI during migration.
 	ActivePods map[types.UID]string `json:"activePods,omitempty"`
+
+	// VolumeStatus contains the statuses of all the volumes
+	// +optional
+	// +listType=atomic
+	VolumeStatus []VolumeStatus `json:"volumeStatus,omitempty"`
+}
+
+// VolumeStatus represents information about the status of volumes attached to the VirtualMachineInstance.
+// +k8s:openapi-gen=true
+type VolumeStatus struct {
+	// Name is the name of the volume
+	Name string `json:"name"`
+	// If the volume is hotplug, this will contain the hotplug status.
+	Hotplug *HotplugStatus `json:"hotplug,omitempty"`
 }
 
+// HotplugStatus represents the hotplug status of the volume
+// +k8s:openapi-gen=true
+type HotplugStatus struct {
```

Comment:
I'd call this HotplugVolumeStatus in case there's another Hotplug type

Reply:
Good idea, I will change it.